functions/src/scripts/add-auto-resolution.ts

```diff
@@ -0,0 +1,58 @@
+// Existing contracts don't have auto resolutions (Time + resolution). Let's add them.
```

Comment:
I take mqp's position that there are particular markets that were intended by their creators to be resolved years into the future, which we'll need to manually fix. I think this is great for the majority of markets though!

Reply:
I'll implement an eMail notification 1 day before the auto resolution to warn creators to change the auto resolution date in case they don't want their markets resolved yet in another PR.

We can think of other notifications for the creators as well since the feature is new.